FileManager/FileManager/ViewController.swift

```diff
@@ -0,0 +1,141 @@
+    var arrfolder : [String] = []
+    var selectName = ""
+    let file = FileManager.default
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        switchVc.setOn(false, animated: false)
+        relodFolder()
+        tableVc.delegate = self
+        tableVc.dataSource = self
+    }
+    
+    @IBAction func buttonfolder(_ sender: Any) {
+        let file = FileManager.default
+        let dirUrl = file.urls(for: .documentDirectory, in: .userDomainMask).first
+        let directory = dirUrl?.appendingPathComponent(textfieldfolder.text!)
+        print(dirUrl?.path)
+        do {
+            try file.createDirectory(at: directory!, withIntermediateDirectories: true , attributes: [:])
+        }catch {
+            print("Somthing wrong")
+        }
+        arrfolder.removeAll()
+        relodFolder()
+        tableVc.reloadData()
+        
+    }
+    func relodFolder() {
+        let file = FileManager.default
+        
+        let dirURL = file.urls(for: .documentDirectory, in: .userDomainMask).first
+        do {
+            let folders = try file.contentsOfDirectory(atPath: dirURL!.path)
+            print(folders)
+            
+            for folder in folders {
+                let test = dirURL?.appendingPathComponent(folder)
+                if test?.hasDirectoryPath == true {
+                    self.arrfolder.append(folder)
+                }else{
+                    print("sorry is not folder\(folder)")
+                }
+                
+            }
+            
+        }catch{
+            print("file")
+        }
+        
+    }
+    func creatDir(dieName:String){
+        guard dieName.isEmpty == false else {return}
+        let fileManager = FileManager.default
+        let dirURL = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first
+        let url = dirURL!.appendingPathComponent(dieName)
+        do {
+            try fileManager.createDirectory(at: url, withIntermediateDirectories: true, attributes: [:])
+            arrfolder.removeAll()
+            relodFolder()
+        }catch {
+            print("ddddd")
+        }
+        
+    }
+    @IBAction func buttondis(_ sender: Any) {
+        guard selectName != "" else {return}
+        guard entertext.text!.isEmpty != true else {return}
+        let file = FileManager.default
+        let dirURL = file.urls(for:.documentDirectory, in: .userDomainMask).first
+        let nameFile = dirURL?.appendingPathComponent(selectName).appendingPathComponent(entertext.text! + ".txt")
+        let content = entertext.text?.data(using: .utf8)
+        file.createFile(atPath: nameFile!.path, contents: content, attributes: [:])
+        
+    }
+    
+    @IBAction func switchVc(_ sender: Any) {
+        if switchVc.isOn == true {
+            view.backgroundColor = .lightGray
+        }else{
+            view.backgroundColor = .systemBrown
+        }
+        
+    }
+    
+    @IBAction func switchesaction(_ sender: Any) {
```

Comment:
you don't use this
@IBAction func switchesaction(_ sender: Any) {
the reason is this code is not associated with the element